src/Microsoft.ML.AutoML/Utils/MLNetUtils/Hashing.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.AutoML
+{
+    internal static class Hashing
```

Comment:
We can't use the existing utility functions in ML.NET because they aren't public, right?

Reply:
they're not but now that I have taken project level dependency I can do BestFriends and nuke the private copies.

Reply:
Yes we can.  I made it work for some of them but it requires some non-trivial changes to make them all go away completely, I filed a work item https://github.com/dotnet/machinelearning/issues/4064 for this.